packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -192,6 +192,8 @@ export const UnreferencedState = {
     Inactive: "Inactive",
     /** The node is ready to be deleted by the sweep phase. */
     SweepReady: "SweepReady",
+    /** The node has been tombstoned. */
+    Tombstone: "Tombstone",
```

Comment:
The nodes are never in this state so I would rather not add it here. SweepReady == Tombstone logically.